src/Toml/BiMap.hs

```diff
@@ -232,6 +254,20 @@ _Array elementBimap = BiMap
     , backward = \(AnyValue val) -> matchArray (backward elementBimap) val
     }
 
+_NonEmpty :: BiMap a AnyValue -> BiMap (NE.NonEmpty a) AnyValue
+_NonEmpty bimap = BiMap (Just . NE.toList) toMaybeNonEmpty >>> _Array bimap
+  where toMaybeNonEmpty [] = Nothing
+        toMaybeNonEmpty x = Just (NE.fromList x)
```

Comment:
This is just [`nonEmpty`](https://hackage.haskell.org/package/base-4.12.0.0/docs/Data-List-NonEmpty.html#v:nonEmpty) function.

Reply:
Thank you, I missed it in the docs. I've never used `NonEmpty` before